packages/twenty-front/src/modules/object-record/record-field/meta-types/input/components/LinksFieldInput.tsx

```diff
@@ -1,93 +1,134 @@
+import { useMemo, useRef, useState } from 'react';
+import styled from '@emotion/styled';
+import { Key } from 'ts-key-enum';
+import { IconPlus } from 'twenty-ui';
+
 import { useLinksField } from '@/object-record/record-field/meta-types/hooks/useLinksField';
-import { FieldInputOverlay } from '@/ui/field/input/components/FieldInputOverlay';
-import { TextInput } from '@/ui/field/input/components/TextInput';
+import { LinkDisplay } from '@/ui/field/display/components/LinkDisplay';
+import { LightIconButton } from '@/ui/input/button/components/LightIconButton';
+import { DropdownMenu } from '@/ui/layout/dropdown/components/DropdownMenu';
+import { DropdownMenuInput } from '@/ui/layout/dropdown/components/DropdownMenuInput';
+import { DropdownMenuItemsContainer } from '@/ui/layout/dropdown/components/DropdownMenuItemsContainer';
+import { DropdownMenuSeparator } from '@/ui/layout/dropdown/components/DropdownMenuSeparator';
+import { MenuItem } from '@/ui/navigation/menu-item/components/MenuItem';
+import { useScopedHotkeys } from '@/ui/utilities/hotkey/hooks/useScopedHotkeys';
+import { useListenClickOutside } from '@/ui/utilities/pointer-event/hooks/useListenClickOutside';
+import { isDefined } from '~/utils/isDefined';
 
-import { FieldInputEvent } from './DateTimeFieldInput';
+const StyledDropdownMenu = styled(DropdownMenu)`
+  left: -1px;
+  position: absolute;
+  top: -1px;
+`;
 
 export type LinksFieldInputProps = {
-  onClickOutside?: FieldInputEvent;
-  onEnter?: FieldInputEvent;
-  onEscape?: FieldInputEvent;
-  onTab?: FieldInputEvent;
-  onShiftTab?: FieldInputEvent;
+  onCancel?: () => void;
```

Comment:
Shouldn't we lift the events here, at least onSubmit, and modify draft values in the dropdown while it's open ?

Reply:
I updated the component and now it uses `onSubmit`. About draft values, how should we use it? Links are persisted on submit so I am not sure if the draft value is needed?